sources/Metal/Support/iTermCharacterSource.m

```diff
@@ -542,6 +542,7 @@ - (void)initializeStateIfNeededWithFont:(CTFontRef)runFont {
         }
     }
     for (int i = 0; i < _numberOfIterationsNeeded; i++) {
+        CGContextRelease(_cgContexts[i]);
```

Comment:
This looks wrong to me because this method only gets to run once in the lifetime of the iTermCharacterSource object. If you can catch a nonnull _cgContexts[i] at this point it's a more serious bug than a simple memory leak.

Reply:
Never mind! I was looking at the wrong branch. The fix is to cherrypick 859a1b783d4dee83006772d19bd509ce23a7fa70 to master.